web/crux/src/app/image/interceptors/image.patch.interceptor.ts

```diff
@@ -27,12 +27,14 @@ export default class ImagePatchValidationPipe extends BodyPipeTransform<PatchIma
         },
       },
       where: {
-        id: value.id,
+        id: versionId,
       },
     })
 
     checkVersionMutability(image.version)
 
-    return PatchImageRequest.fromJSON(value) // it's necassary beacuse of user gets serialized as Long object
+    // TODO(@m8vago, @polaroi8d) validation?
```

Comment:
Please expand a little bit 🙏🏻

Reply:
Should we validate the patch when the user modifies something in the config screen, or save everything, and only validate on when the user want's the use the configuration (for deploy) ?